python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -499,15 +499,19 @@ def split(attrs, inputs, proto_obj):
 
 def _slice(attrs, inputs, proto_obj):
     """Returns a slice of the input tensor along multiple axes."""
+    input_tensor_data = proto_obj.model_metadata.get('input_tensor_data')[0]
+    input_shape = input_tensor_data[1]
     new_attrs = translation_utils._fix_attribute_names(attrs,
                                                        {'axes' : 'axis',
                                                         'ends' : 'end',
                                                         'starts' : 'begin'})
     # onnx slice provides slicing on multiple axis. Adding multiple slice_axis operator
     # for multiple axes from mxnet
     begin = new_attrs.get('begin')
-    end = new_attrs.get('end')
+    end = list(new_attrs.get('end'))
```

Comment:
if `end` does not exist in new_attrs, converting it to a list will throw NoneType error

Reply:
Its a required field in ONNX, will always exist https://github.com/onnx/onnx/blob/master/docs/Changelog.md#slice-1